recipes/sourmash/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: sourmash
+  version: "1.0"
+
+source:
+  fn: sourmash-1.0.tar.gz
+  url: https://pypi.python.org/packages/73/48/6b8d2d1f8930ad028d78a890d53ae752f2d1ac52f763975eefefd75c63fc/sourmash-1.0.tar.gz
+  md5: 25bb61cd9da7b9deba59b6bfe1944f0d
+
+build:
+  # noarch_python: True
+  # preserve_egg_dir: True
+  entry_points:
+    - sourmash = sourmash_lib.__main__:main
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - screed >=0.9
+    - pyyaml >=3.11
+    - libgcc
```

Comment:
Here is the failed build before adding libgcc . 

https://travis-ci.org/phac-nml/bioconda-recipes/jobs/223166549#L1859

Error near the bottom.